packages/react/src/BacktraceClient.ts

```diff
@@ -9,7 +9,7 @@ import { BacktraceConfiguration } from '@backtrace/browser';
 import { BacktraceClientBuilder } from '@backtrace/browser';
 
 export class BacktraceClient extends BacktraceCoreClient {
-    public static instance: BacktraceClient;
+    private static _instance: BacktraceClient;
```

Comment:
Can we only mark this as optional?

```
private static _instance?: BacktraceClient;
```

I know that I've written this in my suggestion without the question mark, I'm sorry.

Reply:
Absolutely! Done.